### DISCUSSION TITLE
Discussion board!

### DISCUSSION THREAD

**Discussion by notnotmelon** [notnotmelon/factorissimo-2-notnotmelon#26] [Announcements]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by paulohsandrade** [comment]
Great surprise, having this updated!
Question: Is it possible for a surface to inherit the special mechanics of **other** planets?

Ideas:
- It would be lovely to have a "greenhouse" that you can actually go in and have Agricultural towers harvesting Gleba trees on Nauvis. 
  - A tech requiring Agricultural Science, after Overgrowth Soil.
  - A recipe that involves both kinds Overgrowth Soils.
  - A "factory floor" tile on the inside in which both kinds of Overgrowth Soils can be placed.

**Reply by notnotmelon** [reply]
I admit this suggestion is now living rent-free in my head.

**Reply by notnotmelon** [reply]
This is now added.

**Reply by paulohsandrade** [reply]
Oh wow, the double spore generation inside is quite a nice balance change. Yeah, now you can have your trees closer, but you better be prepared to reinforce those defenses! Loved it.

**Comment by varkarrus** [comment]
Is there no way of having conbots only go to factories as needed? Request as many conbots into a factory as there are ghosts, and then give the factories logistic trash slots so that bots can then take the bots and excess items away?

**Reply by notnotmelon** [reply]
"As many as needed" could lead to factories requesting thousands of bots. Consider that each entity must request its modules as well.
This also causes problems for when you would like to edit the factory after its built. For example by using the upgrade planner.

As for the second idea, its possible for the user to build some storage chest->active chest combo by using factory chest connections.
![image](https://github.com/user-attachments/assets/9109b93f-95d2-4449-b38e-2d74b17683dc)
![image](https://github.com/user-attachments/assets/f8e8ba7b-9f61-4f15-a4a3-6a0c3e7c0de8)
This would allow a mechanism to automatically trash excess items, even in recursive factory floors.

**Reply by varkarrus** [reply]
I wouldn't mind it requesting thousands of bots if those bots could then return to the main logistics network when they're done

**Reply by notnotmelon** [reply]
This is fixed now.

**Comment by rsstst** [comment]
do you have any plan to make something like factorissimo inception/playthrough and probably with mods compatibility?

**Reply by notnotmelon** [reply]
I don't. But its easy to make it with my remote API.

**Comment by JehuLLL** [comment]
I don't understand the significance of this alarm.I have already deployed the construction drone, but the alarm still cannot be eliminated. 
![警报](https://github.com/user-attachments/assets/4ac081bc-904f-46ff-949d-36242e7a4874)

**Reply by notnotmelon** [reply]
This is removed now.
